src/components/Tab.tsx

```diff
@@ -0,0 +1,41 @@
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles'
+import Tab from '@material-ui/core/Tab'
+
+interface StyledTabProps {
+  label: string
+}
+
+export default withStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      textTransform: 'none',
+      minWidth: 72,
+      backgroundColor: 'transparent',
+      fontWeight: theme.typography.fontWeightRegular,
+      marginRight: theme.spacing(4),
+      fontFamily: [
+        '-apple-system',
+        'BlinkMacSystemFont',
+        '"Segoe UI"',
+        'Roboto',
+        '"Helvetica Neue"',
+        'Arial',
+        'sans-serif',
+        '"Apple Color Emoji"',
+        '"Segoe UI Emoji"',
+        '"Segoe UI Symbol"',
+      ].join(','),
```

Comment:
I was thinking if it would be an improvement to extract this to the theme or some other place as a constant. I assume this is the default many times and will be reused.